daemon/api.go

```diff
@@ -925,6 +907,25 @@ func appIconGet(c *Command, r *http.Request) Response {
 	return FileResponse(path)
 }
 
+// FXIME: can we kill this? all icons are always availble via the canonical
+//        /1.0/icons/$name.$origin/icon
+func metaIconGet(c *Command, r *http.Request) Response {
```

Comment:
None whatsoever: there are two situations under which these endpoints are called: one, when referenced from the snap details object (e.g. the output of `GET /1.0/packages/hello-world.canonical`), and two, when you're poking at the system with a stick to see what breaks.